src/deployers/PlatformPluginsDeployer.cpp

```diff
@@ -1,24 +1,50 @@
 // library headers
 #include <linuxdeploy/core/log.h>
-#include <boost/filesystem.hpp>
 
 // local headers
 #include "PlatformPluginsDeployer.h"
+#include "linuxdeploy/util/util.h"
 
 using namespace linuxdeploy::plugin::qt;
 using namespace linuxdeploy::core::log;
 
 namespace bf = boost::filesystem;
 
+PlatformPluginsDeployer::PlatformPluginsDeployer(std::string moduleName,
+                                                 core::appdir::AppDir& appDir,
+                                                 bf::path qtPluginsPath,
+                                                 bf::path qtLibexecsPath,
+                                                 bf::path installLibsPath,
+                                                 bf::path qtTranslationsPath,
+                                                 bf::path qtDataPath):
+                                                 BasicPluginsDeployer(std::move(moduleName),
+                                                                      appDir,
+                                                                      std::move(qtPluginsPath),
+                                                                      std::move(qtLibexecsPath),
+                                                                      std::move(installLibsPath),
+                                                                      std::move(qtTranslationsPath),
+                                                                      std::move(qtDataPath)) {
+    // check if the platform plugins are set in env
```

Comment:
Not sure whether it's a great idea to implement a custom constructor. Of course, software design wise, that's probably the way to go, given that we need to create that list below. However, it adds a fair amount of boilerplate code that will have to be maintained. What do you think about moving the evaluation of the environment variable into `deploy()`? I am undecided which version would be better in the long term.

Reply:
It doesn't seem like `platformToDeploy` is used at all outside of the `deploy` function, and that function is only called once per instance, so I think it could just be a local variable.